buildbot/buildbot/state.py

```diff
@@ -0,0 +1,12 @@
+# Define various states workers can be in
+
+UPLOAD = "Uploaded",
+UNPACK = "Unpacking",
+UNPACK_FINISH = "Unpacked",
+COMPILE = "Compiling",
+COMPILE_FINISH = "Compiled",
+HLS = "HLS",
+HLS_FINISH = "HLSed",
+RUN = "Running",
+DONE = "Done"
+FAIL = "Failed"
```

Comment:
Let's keep these strings as all lower-case, please. (They're meant as internal status strings, not part of a polished UI. If we want to make the UI version of these strings better, let's make those separate. Keeping the same names as before will also make currently-terminated job data we have lying around still look right. Also, capitalized CSS class names look weird. :smiley:)

Also, these lines do not need trailing commas.

Reply:
Is there a way to easily import a map from internal string to UI strings in Flask templates?

Reply:
A Python dictionary? :smiley: You can do `{{ state_strings[state] }}`, for example.

Reply:
Can I also write `import state` in the template? I understand how to use python dicts, but wanted to keep all state data in state.py.

Reply:
There are two good ways to do this: pass along `state_strings` as an additional parameter to the template (from `server.py`), or set up the templating engine so that `state_strings` is in the [global namespace](http://jinja.pocoo.org/docs/2.10/api/#the-global-namespace).

Reply:
But, the human-readable names of states might best belong in `server.py` because it's bowser-UI-related…

Reply:
Fixed.